gammapy/detect/test_statistics.py

```diff
@@ -360,16 +367,20 @@ def compute_ts_map(counts, background, exposure, kernel, mask=None, flux=None,
 
     # Set TS values at given positions
     j, i = zip(*positions)
+    TS = np.ones(counts.shape) * np.nan
     amplitudes = np.zeros(TS.shape)
     niter = np.zeros(TS.shape)
     TS[j, i] = [_[0] for _ in results]
     amplitudes[j, i] = [_[1] for _ in results]
     niter[j, i] = [_[2] for _ in results]
-    return TSMapResult(ts=TS, amplitude=amplitudes,
+
+    # Compute edge cleaned sqrt TS map
+    sqrt_TS = np.where(TS > 0, np.sqrt(TS), -np.sqrt(-TS))
```

Comment:
Many users will wonder what a negative sqrt_TS means.
Can you give this formula in the function docstring or even make a section explaining it in the high-level docs and then reference to it from function docstrings that use this?